bionic/protocols.py

```diff
@@ -642,7 +642,7 @@ def write(self, value, path):
         self._protocol_for_value(value).write(value, path)
 
     def __repr__(self):
-        return f"CombinedProtocol{tuple(self._subprotocols)!r}"
+        return "CombinedProtocol(...)"
```

Comment:
Oh, good question. I found the original version super long and not very informative, because we always use the same combination of protocols. It just takes up a lot of screen space anytime you print a protocol (because most things use the same default protocol, which is a big CombinedProtocol).